### DISCUSSION TITLE
How to use neutralinojs with Next.js?

### DISCUSSION THREAD

**Discussion by 6174** [neutralinojs/neutralinojs#1070] [Q&A]
I am trying find a solution to create a desktop for a app (https://doodleboard.pro) which is based on Next.js.   I don't want to use electron because it's too heavy. 

I think neutralinojs looks good for my need, but I don't figure out a way to easily integrate my next.js project with neutralinojs. 

So, I'am asking for help, are there some suggestions or ready-made examples?

**Comment by tungnnt309** [comment]
I think I can give you some suggestions:
- try this guide to create a neutralinojs project from scratch: [guide](https://neutralino.js.org/docs/getting-started/using-frontend-libraries#using-any-frontend-library), you can create a totally new project by CLI, not necessary to run in your existing project
- from your existing nextjs project, try to build then export with `next build` and `next export` to get the `out` folder as default after nextjs building process
- copy your `out` folder to neutralinojs project, then edit your `documentRoot` in the neutralinojs config file

**Comment by sahaskamble** [comment]
did you get any success because I can't use static export for nextjs ?
